build_system_fr.md

```diff
@@ -101,7 +101,7 @@ $ head debian/changelog
 $ git yunobump x.y.z-p
 
 # Push the branch state AND the tags to the remote repository
-$ git push origin --tags testing:testing
+$ git push origin --tags testing : testing
```

Comment:
I'm not going to re-read what I've done today, but I'll review it later :)  
I will note errors I created :+1:
On 16-03-30 06:05:38, opi wrote:
>> @@ -101,7 +101,7 @@ $ head debian/changelog
>>  $ git yunobump x.y.z-p
>>
>>  # Push the branch state AND the tags to the remote repository
>> -$ git push origin --tags testing:testing
>> +$ git push origin --tags testing : testing
>
>git command will not work with spaces around colon
>
>---
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/YunoHost/doc/pull/360/files/fe5af54348a5c9b4a2f8455f8fb8ad0e0f2b3e68#r57884208